dataland-frontend/tests/component/components/resources/frameworkDataSearch/gdvVoeb/GdvVoebPanel.cy.ts

```diff
@@ -0,0 +1,227 @@
+    const preparedFixture = getPreparedFixture("Gdv-dataset-with-no-null-fields", preparedFixtures);
+    const gdvData = preparedFixture.t;
+
+    cy.intercept("/api/data/gdv/mock-data-id", {
+      companyId: companyId,
+      reportingPeriod: preparedFixture.reportingPeriod,
+      data: gdvData,
+    } as CompanyAssociatedDataGdvData);
+    mountGDVFrameworkFromFakeFixture(DataTypeEnum.Gdv, configForGdvVoebPanelWithOneListForBaseDataPointComponent, [
+      preparedFixture,
+    ]);
+
+    const listData: Array<BaseDataPointString> = preparedFixture.t.allgemein?.esgBerichte?.aktuelleBerichte;
+    cy.get("span").contains("Aktuelle Berichte");
+    cy.get("a").should("have.class", "link").click();
+
+    cy.get("span").contains("Beschreibung des Berichts");
+    for (const oneListElement of listData) {
+      cy.get("div").contains(oneListElement.value);
+      cy.get("div").contains(oneListElement.dataSource?.fileName as string);
+    }
+    cy.get('span[data-test="Report-Download-Policy"]').should("exist");
+  });
+});
+
+/**
+ *
+ * Mounts the MultiLayerDataTableFrameworkPanel with the given dataset for the GDV framework
+ * @param frameworkIdentifier the identifier of the framework
+ * @param displayConfiguration the MLDT display configuration for the current framework
+ * @param fixtureDatasetsForDisplay the datasets from the fixtures to mount
+ * @param companyId company ID of the mocked requests
+ * @param reviewMode toggles the reviewer mode
+ * @returns the component mounting chainable
+ */
+function mountGDVFrameworkFromFakeFixture(
+  frameworkIdentifier: DataTypeEnum,
+  displayConfiguration: MLDTConfig<GdvData>,
+  fixtureDatasetsForDisplay: Array<FixtureData<GdvData>>,
+  companyId = "mock-company-id",
+  reviewMode = false,
+): Cypress.Chainable {
+  const convertedDataAndMetaInformation: Array<DataAndMetaInformation<GdvData>> = fixtureDatasetsForDisplay.map(
+    (it, idx) => {
+      const metaInformation: DataMetaInformation = {
+        dataId: `data-id-${idx}`,
+        companyId: companyId,
+        dataType: frameworkIdentifier,
+        uploadTime: 0,
+        reportingPeriod: it.reportingPeriod,
+        qaStatus: "Accepted",
+        currentlyActive: true,
+      };
+      return {
+        data: it.t,
+        metaInfo: metaInformation,
+      };
+    },
+  );
+
+  return mountMLDTForGdvPanel(
+    frameworkIdentifier,
+    displayConfiguration,
+    convertedDataAndMetaInformation,
+    companyId,
+    reviewMode,
+  );
+}
+
+/**
+ * Mounts the MultiLayerDataTableFrameworkPanel with the given dataset
+ * @param frameworkIdentifier the identifier of the framework whose datasets should be displayed
+ * @param displayConfiguration the MLDT display configuration
+ * @param datasetsToDisplay datasets to mount
+ * @param companyId company ID of the mocked requests
+ * @param reviewMode toggles the reviewer mode
+ * @returns the component mounting chainable
+ */
+export function mountMLDTForGdvPanel(
```

Comment:
Don't we have similiar functions for the other frameworks?

Reply:
Do you mean `mountMLDTFrameworkPanelFromFakeFixture()` ?
That one probably cannot be used here out of the box, because it expects the framework to be of type:
`Framework extends keyof FrameworkDataTypes`, and `FrameworkDataTypes` in the file `FrameworkDataTypes.ts` does not (and should not) contain gdv.

Gdv is handled in another list of frameworks, which is the auto-filled `BaseFrameworkRegistryImports.ts`. All devtools-supported frameworks will apear there.
So I assume @marcelrasbachdfine had to write a new function here.

Correct me if I am wrong @MarcTM01

Reply:
Ok fair point. But when we introduce the next framework to the toolbox and add component test, we should generalize this function here, similar to the way we do it right now for the existing frameworks.

Reply:
Closed for now